src/components/Swap/Summary/Details.tsx

```diff
@@ -68,9 +68,8 @@ export default function Details({ trade, slippage, impact }: DetailsProps) {
       rows.push([t`Price impact`, impact.toString(), impact.warning])
     }
 
-    if (lpFeeAmount) {
-      const parsedLpFee = formatCurrencyAmount(lpFeeAmount, 6, i18n.locale)
-      rows.push([t`Liquidity provider fee`, `${parsedLpFee} ${inputCurrency.symbol || currencyId(inputCurrency)}`])
+    if (gasUseEstimateUSD) {
+      rows.push([t`Network fee`, `~$${gasUseEstimateUSD.toFixed(2)}`])
```

Comment:
this would be a candidate for a format function

Reply:
Completely agree - AFAIK this is a follow-up for widget, generally. Right now https://github.com/uniswap/interface has a `formatDollar` function, but https://github.com/uniswap/widgets does not yet.